AnnService/inc/Core/SPANN/ExtraFullGraphSearcher.h

```diff
@@ -272,7 +273,37 @@ namespace SPTAG
                         LOG(Helper::LogLevel::LL_Error, "File %s read bytes, expected: %zu, acutal: %llu.\n", m_extraFullGraphFile.c_str(), totalBytes, numRead);
                         exit(-1);
                     }
-                    ProcessPosting(m_vectorInfoSize)
+                    // decompress posting list
+                    char* p_postingListFullData = buffer + listInfo->pageOffset;
+                    if (m_enableDataCompression)
```

Comment:
too many repeated code. Could we move the decompression to marco or use a function to reduce the repeated code?

Reply:
> too many repeated code. Could we move the decompression to marco or use a function to reduce the repeated code?

refactored